Flake in e2e test packages

Here for future reference. I've only seen this once on locally on OSX.

```
Running legacy e2e test packages
checking deps: release
all deps built
+ yarn test
yarn run v1.16.0
warning package.json: No license field
$ bazel test //...
Starting local Bazel server and connecting to it...
INFO: Analyzed 5 targets (93 packages loaded, 993 targets configured).
INFO: Found 1 target and 4 test targets...
FAIL: //:version_yarn_install_test (see /private/var/tmp/_bazel_greg/53a2d0989782ca8eaeab3d865af2e9c0/execroot/internal_e2e_packages/bazel-out/darwin-fastbuild/testlogs/version_yarn_install_test/test.log)
INFO: From Testing //:version_yarn_install_test:
==================== Test output for //:version_yarn_install_test:
Error: //:version_yarn_install_test_bin cannot find module './test_version' required by '/private/var/tmp/_bazel_greg/53a2d0989782ca8eaeab3d865af2e9c0/sandbox/darwin-sandbox/3/execroot/internal_e2e_packages/bazel-out/darwin-fastbuild/bin/version_yarn_install_test.runfiles/internal_e2e_packages/yarn_install.spec.js'
  looked in:
    built-in, relative, absolute, nested node_modules - Error: Cannot find module './test_version'
    runfiles - Error: Cannot find module '/private/var/tmp/_bazel_greg/53a2d0989782ca8eaeab3d865af2e9c0/sandbox/darwin-sandbox/3/execroot/internal_e2e_packages/bazel-out/darwin-fastbuild/bin/version_yarn_install_test.runfiles/test_version'
    node_modules attribute (internal_e2e_packages_yarn_install/node_modules) - Error: Cannot find module '/private/var/tmp/_bazel_greg/53a2d0989782ca8eaeab3d865af2e9c0/sandbox/darwin-sandbox/3/execroot/internal_e2e_packages/bazel-out/darwin-fastbuild/bin/version_yarn_install_test.runfiles/internal_e2e_packages_yarn_install/node_modules/test_version'

    at Function.module.constructor._resolveFilename (version_yarn_install_test_bin_loader.js:472:17)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (yarn_install.spec.js:1:1)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
================================================================================
INFO: Elapsed time: 12.747s, Critical Path: 0.40s
INFO: 8 processes: 8 darwin-sandbox.
INFO: Build completed, 1 test FAILED, 25 total actions
//:npm_determinism_test                                                  PASSED in 0.2s
//:version_npm_install_test                                              PASSED in 0.2s
//:yarn_determinism_test                                                 PASSED in 0.2s
//:version_yarn_install_test                                             FAILED in 0.2s
  /private/var/tmp/_bazel_greg/53a2d0989782ca8eaeab3d865af2e9c0/execroot/internal_e2e_packages/bazel-out/darwin-fastbuild/testlogs/version_yarn_install_test/test.log

Executed 4 out of 4 tests: 3 tests pass and 1 fails locally.
INFO: Build completed, 1 test FAILED, 25 total actions
error Command failed with exit code 3.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

--- a/.bazelrc
+++ b/.bazelrc
@@ -33,3 +33,7 @@ import %workspace%/third_party/github.com/bazelbuild/bazel-toolchains/bazelrc/ba
 # Remote instance, borrow the one used by Angular devs
 build:remote --remote_instance_name=projects/internal-200822/instances/default_instance
 build:remote --project_id=internal-200822
+
+# Our integration tests rely on accessing public registries which can fail
+# Let's give all failing tests a second try to mostly paper over flakiness
+test --flaky_test_attempts=2

